src/components/Subscriptions.js

```diff
@@ -7,7 +7,21 @@ import createBillingAgreement from '../helpers/createBillingAgreement'
 import queryString from 'query-string'
 import '../assets/Subscriptions.css'
 
+let membership = props => {
```

Comment:
This is clearly something that should come from the backend. But we will do that in a different story